en/docs/administer/key-managers/configure-auth0-connector.md

```diff
@@ -0,0 +1,302 @@
+# Configure Auth0 as a Key Manager
+
+It is possible to integrate the WSO2 API Manager with an external Identity and Access Management server (IAM) using the Auth0 OAuth Authorization Server to manage the OAuth clients and tokens that are required by WSO2 API Manager. WSO2 API Manager has inbuilt support to consume APIs exposed by Auth0 OAuth.
+
+Follow the instructions below to configure Auth0 as a third-party Key Manager:
+
+!!! info
+    For more information, see the [Getting Started Guide, which is under the official Auth0 documentation](https://auth0.com/docs/get-started).
+
+## Step 1 - Configure Auth0
+
+1.  Create an Auth0 account. Get the URL for the tenant and then sign in to the dashboard.
+
+   [![auth0 dashboard]({{base_path}}/assets/img/administer/auth0-dashboard.png)]({{base_path}}/assets/img/administer/auth0-dashboard.png)
+
+2.  Then you need to create an application to use the management API. Then need's to allow that application to use the management API.
+
+    [![auth0 management api]({{base_path}}/assets/img/administer/auth0-management-api.png)]({{base_path}}/assets/img/administer/auth0-management-api.png)
+
+    [![auth0 new application]({{base_path}}/assets/img/administer/auth0-new-application.png)]({{base_path}}/assets/img/administer/auth0-new-application.png)
+    
+    [![auth0 permission to use app]({{base_path}}/assets/img/administer/auth0-permission-to-use-app.png)]({{base_path}}/assets/img/administer/auth0-permission-to-use-app.png)
+   
+    Make sure you have granted all the permissions to Create, Manage Apps and Resource servers. 
+   
+3.  Create a custom API by navigating to APIs as below.
+
+    [![auth0 create custom api]({{base_path}}/assets/img/administer/auth0-create-custom-api.png)]({{base_path}}/assets/img/administer/auth0-create-custom-api.png)
+
+    <a name="step13"></a>
+    
+4.  Create a new scope named `default` for the API by navigating to permissions tab in the custom API.    
+
+    [![auth0 create scope]({{base_path}}/assets/img/administer/auth0-create-scope.png)]({{base_path}}/assets/img/administer/auth0-create-scope.png)
+
+## Step 2 - Configure WSO2 API Manager
+
+1.  Log into the admin portal of the API Manager and add a new Key Manager.
```

Comment:
"Log in to" should be "Sign in to". We use "log in" when referring to logging in to a server and we use "sign in" when referring to UI consoles.